controllers/apply/awards-for-all/fields/senior-contact-role.js

```diff
@@ -73,9 +73,12 @@ module.exports = function(locale, data) {
             if (isCharityOrCompany && projectCountry !== 'scotland') {
                 result.push(
                     localise({
-                        en: oneLine`Your senior contact must be listed as a
-                            member of your organisation's board or committee
-                            with the Charity Commission/Companies House.`,
+                        en: oneLine`If you're a company, your senior contact
+                            must be listed as a member of your organisation's
+                            board with Companies House.
+                            If you're an unincorporated Charity, your Senior
+                            contact should be listed as a Trustee on the
+                            Charities regulator website.`,
```

Comment:
Think Regulator should be capitalised, eg. https://www.charitiesregulator.ie/en

(could/should this be a link to that site, too?)

Reply:
I did a quick review of the content (mainly adding subheadings for scannability and also reducing excess text). See what you think: 

**If you're a company**
Your senior contact needs to be listed as one of your board members on Companies House. 

**If you're a registered charity (that's not also a company)**
Your senior contact needs to be listed as one of your trustees with your charity regulator. 

An alternative suggestion - is it too complex to make each subheading appear only to the relevant types of organisations? So that we can reduce text on-screen. 

Then it coudl be just something like 'As you're a company, you senior contact needs to be listed as one of your board memebrs on Companies House.' and so on

Reply:
No, that's simpler if anything, let's go with separate text

Reply:
Great! So:

'As you're a company, your senior contact needs to be listed as one of your board members on Companies House.'

And:

'As you're a registered charity (that's not also a company), your senior contact needs to be listed as one of your trustees with your charity regulator.'

Might be good to get Kirsten to do a review too?